test/conftest.py

```diff
@@ -1,13 +1,54 @@
 # pytest fixtures available in tests
-
 from pathlib import Path
 import json
 from copy import deepcopy
 import pytest
+import jsonschema
+
+import tools.validator as validator_
+from tools.bundle_schema import bundle_schema
+from tools.downgrade_schema_to_draft07 import downgrade_schema_to_draft07
```

Comment:
Why might we need to downgrade to draft07?

Reply:
Most libraries are still stuck with draft07, so for widest support this is unfortunately a requirement. Someone on the OGC repo also mentioned that, for some Java piece that supports CovJSON.

Reply:
Are there any issues to be aware of connected with the downgrading? Any features that will be missing, or constructs we should avoid?

Reply:
Running the tests against both the downgraded schemas and the native ones makes sure that both are consistent. There may be some constructs we don't use yet that can't be downgraded, but we'll notice it once it happens.